x-pack/plugins/security_solution/public/detections/components/rules/description_step/index.tsx

```diff
@@ -152,6 +152,7 @@ export const addFilterStateIfNotThere = (filters: Filter[]): Filter[] => {
 };
 
 /* eslint complexity: ["error", 21]*/
+// eslint-disable-next-line complexity
```

Comment:
One or the other -- no need for both here. We try to keep it in check with the count, but this just needs to be refactored at this point.